packages/react-mutation-mapper/src/component/clinvar/ClinVarSummary.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+    DefaultTooltip,
+    pluralize,
+    TruncatedText,
+} from 'cbioportal-frontend-commons';
+import { getClinVarId } from 'cbioportal-utils';
+import { ClinVar, MyVariantInfo } from 'genome-nexus-ts-api-client';
+import _ from 'lodash';
+import * as React from 'react';
+
+export type ClinVarSummaryProps = {
+    myVariantInfo?: MyVariantInfo;
+};
+
+export enum ClinVarOrigin {
+    GERMLINE = 'germline',
+    SOMATIC = 'somatic',
+}
+
+export type RcvData = {
+    origin: string;
+    evidences: {
+        clinicalSignificance: string;
+        count: number;
+    }[];
+};
+
+export type RcvCountMap = {
+    [origin: string]: { [clinicalSignificance: string]: number };
+};
+
+export function getRcvCountMap(clinVar: ClinVar): RcvCountMap {
+    const filteredRcv = clinVar.rcv.filter(
+        rcv =>
+            rcv.origin === ClinVarOrigin.GERMLINE ||
+            rcv.origin === ClinVarOrigin.SOMATIC
+    );
+
+    // first map by origin, then count evidence number for each origin group
+    const rcvMap = _.groupBy(filteredRcv, d => d.origin);
+
+    return _.mapValues(rcvMap, rcvs =>
+        _.countBy(rcvs, rcv => rcv.clinicalSignificance)
+    );
+}
+
+export function getRcvData(rcvCountMap: RcvCountMap): RcvData[] {
+    return _.map(rcvCountMap, (clinicalSignificanceCountMap, origin) => ({
+        origin,
+        evidences: _.map(
+            clinicalSignificanceCountMap,
+            (count, clinicalSignificance) => ({ clinicalSignificance, count })
+        ),
+    }));
+}
+
+export function formatClinicalSignificanceText(rcvData: RcvData[]) {
+    return _.uniq(
+        _.flatten(
+            rcvData.map(d => d.evidences.map(e => e.clinicalSignificance))
+        )
+    ).join(', ');
+}
+
+export const ClinVarRcvInterpretation = (props: {
+    rcvData: RcvData[];
+    className?: string;
+}) => {
+    return (
+        <div className={props.className}>
+            {props.rcvData.map(d => (
+                <div key={d.origin}>
+                    <strong>{`${_.upperFirst(d.origin)}: `}</strong>
+                    {d.evidences
+                        .map(
+                            e =>
+                                `${e.clinicalSignificance} (${
+                                    e.count
+                                } ${pluralize('evidence', e.count)})`
+                        )
+                        .join(', ')}
+                </div>
+            ))}
+        </div>
+    );
+};
+
+const ClinVarSummary = (props: ClinVarSummaryProps) => {
+    const clinVar = props.myVariantInfo
+        ? props.myVariantInfo.clinVar
+        : undefined;
+
+    if (!clinVar) {
+        return (
+            <DefaultTooltip
+                placement="topRight"
```

Comment:
Probably good to move this to left
![image](https://user-images.githubusercontent.com/16869603/104642846-717d4a00-5679-11eb-9eb5-8c0225e53ce1.png)